am-lib/src/test/java/integration/uk/gov/hmcts/reform/amlib/AccessManagementServiceIntegrationTest.java

```diff
@@ -0,0 +1,28 @@
+package integration.uk.gov.hmcts.reform.amlib;
+
+import org.junit.Test;
+
+import java.util.UUID;
+
+import static org.assertj.core.api.Java6Assertions.assertThat;
+
+@SuppressWarnings("PMD")
```

Comment:
Why the suppression? Is it something that will likely happen in other integration tests? If it is than would be better to suppress the rule for all integration tests rather than do that for each (I've done something like that for RestAssuredTests, so feel free to talk to me about that)

Reply:
Fixed